templates/donation_successful.html

```diff
@@ -1,5 +1,194 @@
+
+<style>
+	html, body, main
+	{
+		height:100%;
+		background-color: #00a850;
+	}
+	header
+	{
+		position: absolute;
+		top: 0;
+		left: 0;
+		right: 0;
+	}
+	#page 
+	{
+		/* The image used */
+		background-image: url("../static/img/donation_successful_background.jpg");
+
+		/* Full height */
+		/* Center and scale the image nicely */
+		background-position: center;
+		background-repeat: no-repeat;
+		background-size: cover;
+
+		position: absolute;
+		top: 0;
+		right: 0;
+		bottom: 0;
+		left: 0;
+
+		margin-top: 80px;
+
+	}
+	#text
+	{
+		display:inline-block;
+		position: absolute;
+		left:30%;
+		top:35%;
+		
+	}
+
+	h1
+	{
+		font-size: 35px;
+		color:white;
+		margin-bottom: 20px;
+	}
+	.link-button
+	{
+		color:white;
+		border-color:white;
+		margin-bottom: 20px;
+	}
+	.fa
+	{
+		color:white;
+	}
+	h4, p, h6
+	{
+		color:white;
+		margin-bottom: 20px;
+	} 
+	#text .icon-list
+	{
+		margin-bottom: 90px;
+		position: relative !important;
+		
+	}
+	#text .icon-list li
+	{
+		margin-right:25px;
+	}
+</style>
+
+</head>
 <body>
-  <h2>Thanks, you paid <strong>${{ "%.2f" % (0.01*amount)}}</strong>!</h2>
+	<main class="cd-main">
+		<header>
+			<nav class="cd-main-nav-wrapper">
+
+				<ul class="cd-main-nav">
+					<li id="main-logo">
+						<img class="arm-nav-logo" src="../static/img/arm_banner.png" alt="Archangel Raphael's Mission">
+					</li>
+
+                    <!-- <li><a href="#0">Blog</a></li>
+                    <li><a href="http://bit.ly/join_us_banner" target='_blank'>Join Us</a>
+                    <li><a href="stripe donate!">Donate</a>
+                    -->
+
+                    <li>
+                    	<!-- Connect button -->
+                    	<a href="#0" class="cd-subnav-trigger"><span>Connect</span></a>
+
+                    	<ul class="icon-list">
+                    		<li class="go-back"><a href="#0">Menu</a></li>
+
+                    		<li><a href="https://www.facebook.com/armnewbrunswick/" target="_blank"><i class="fa fa-facebook fa-2x" aria-hidden="true"></i></a></li>
+                    		<li><a href="https://twitter.com/armnewbrunswick" target="_blank"><i class="fa fa-twitter fa-2x" aria-hidden="true"></i></a></li>
+                    		<li><a href="https://www.instagram.com/armnewbrunswick/" target="_blank"><i class="fa fa-instagram fa-2x" aria-hidden="true"></i></a></li>
+
+                    		<li><a href="https://www.snapchat.com/add/ARMNewBrunswick" target="_blank"><i class="fa fa-snapchat fa-2x" aria-hidden="true"></i></a></li>
+
+                    		<li><a href="mailto:info@armnewbrunswick.org?Subject=How%20Can%20I%20Help?" target="_blank"><i class="fa fa-envelope-o fa-2x" aria-hidden="true"></i></a></li>
+
+                    	</ul>
+                    </li>
+                </ul> <!-- .cd-main-nav -->
+            </nav> <!-- .cd-main-nav-wrapper -->
+
+            <a href="#0" class="cd-nav-trigger">Menu<span></span></a>
+        </header>
```

Comment:
I think we can leverage jinja templating and avoid repeating so much html in multiple places. 

put this into a header.html and use jinja to `include` that file here and on `index.html`.

That way we only have to maintain one copy of how the header looks.